resource-managers/yarn/src/test/scala/org/apache/spark/deploy/yarn/YarnAllocatorSuite.scala

```diff
@@ -394,7 +395,7 @@ class YarnAllocatorSuite extends SparkFunSuite with Matchers with BeforeAndAfter
   }
 
   test("window based failure executor counting") {
-    sparkConf.set("spark.yarn.executor.failuresValidityInterval", "100s")
+    sparkConf.set(EXECUTOR_ATTEMPT_FAILURE_VALIDITY_INTERVAL_MS, 100L)
```

Comment:
It was `100s` so 100 sec and its unit is milliseconds (because of the suffix _MS) so now `100 * 1000L`

Reply:
Actually in this case it's better to set the string directly to "100s" as before.

Reply:
I have checked the code after this line and the clock advanced in the same way:
https://github.com/apache/spark/blob/ae205b1d8ee9bc7690205783c2bf1397f492f7b3/resource-managers/yarn/src/test/scala/org/apache/spark/deploy/yarn/YarnAllocatorSuite.scala#L418-L425

So for me using the same style within a method a bit easier to follow but of course both works.

Reply:
@wangjiaochun missing spaces around operator: "100*1000L" => "100 * 1000L"